.github/workflows/ci.yml

```diff
@@ -80,21 +80,19 @@ jobs:
         name: Codestyle
         run: |
           cd src/
-          pycodestyle
-  whitespace:
+          pycodestyle --max-line-length=88 --ignore=E121,E123,E126,E226,E24,E704,W503,W504,E203
```

Comment:
Do we still need this?

Reply:
While there is certainly a substantial overlap between the things black and pycodestyle check, pycodestyle still has a few additions, which is why I didn't delete it right away.
They include, e.g., "‘async’ and ‘await’ are reserved keywords starting with Python 3.7" ([full list](https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes)).

I'd be in favor of replacing it with [flake8](https://github.com/PyCQA/flake8) which is a wrapper for various other tools or removing it completely.